__tests__/articles.test.js

```diff
@@ -32,6 +32,25 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("200 should return articles filtered by given query topic", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles.length).toBe(1);
```

Comment:
It would be suitable here to check that the topic is definitely what you expect it to be, rather than just checking how many have been returned